frontend/public/tenants/tenant.json

```diff
@@ -10,5 +10,7 @@
     "title": "",
     "heading": "",
     "body": ""
-  }
+  },
+  "layers" : [],
```

Comment:
On a different PR I noticed the auto-formatting "fixed" this for me. Maybe fix it here to avoid other further conflicts? thanks!

If not obvious this should be:

```json
"layers": []  # no space before colon
```